packages/framework/synthesize/src/dependencyContainer.ts

```diff
@@ -78,6 +78,27 @@ export class DependencyContainer<TMap> implements IFluidDependencySynthesizer {
         }
         return false;
     }
+    /**
+     * @deprecated - Needed for back compat
```

Comment:
can you add some context on the deprecation/open a related issue to remove it for reals in the future

Reply:
created #9078. i'll also add more context